ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -27,7 +27,14 @@ import SwiftUI
 ///
 @objc public class MSFListCellState: NSObject, ObservableObject, Identifiable {
     public var id = UUID()
-    @objc @Published public var leadingView: UIView?
+    @Published public var leadingUIView: AnyView?
+    @objc public var leadingView: UIView? {
```

Comment:
As a separate task/PR after you merge this one, can you also do the same approach with all of the UIView properties of the state here?
I think it's a good pattern to support both the pure SwiftUI and the UIKit environments.